### ISSUE TITLE
Convert Gauss1D, Drude1D to Area-Gauss/Drude-1D

### ISSUE THREAD

**Issue by els1** [PAHFIT/pahfit#181]
Instead of using astropy's Gauss1D/Drude1D, use  Area-Gauss/Drude-1D for the modelling (i.e. based on integrated strength).

**Comment by jdtsmith** [comment]
What do you think about `Power` instead of `Area`?  That better conveys the integrated intensity of an f_nu shape.

**Comment by karllark** [comment]
Hummm...I prefer Area as that conveys what we are doing to my brain.  But maybe I'm not understanding or in a minority.  Just integration = area under the curve in a math sense.

**Comment by els1** [comment]
See also #155

**Comment by ThomasSYLai** [comment]
It's good we pass area (power) to the fitter so that different features can be tied easily during the fit.  But it'd be useful to keep a function that can directly output heights (amplitudes) of the PAH and line features (if needed) for training purposes. A lot of times it's easy and straightforward to see the height of a feature in the spectrum but less intuitive for an area.

**Comment by jdtsmith** [comment]
Yeah I've thought the same thing.  Like for the "guess" stage of the fit, it's easy to "guess" amplitude by eye, not so easy with power.  In my new model context I have function for Drude/Gaussian that can take either power of amplitude.  I also internally use "scaled power", which is a little easier to reason about, and takes care of the "big exponent" problem:
 
```
  Scaled Power is power scaled into the flux density units of the
    input spectrum (e.g. mJy, MJy/sr, etc.):

      SP = P lam_0/c

    An interpretation of scaled power: if a feature has a FWHM similar
    to its central wavelength, the scaled power is approximately the
    feature's peak amplitude.  Scaled power is used internally for
    fitting, so as to avoid large mismatch in numeric scale between
    powers and feature amplitudes.
```
